worlds/AutoWorld.py

```diff
@@ -527,7 +537,7 @@ def remove(self, state: "CollectionState", item: "Item") -> bool:
         return False
 
     # following methods should not need to be overridden.
-    def create_filler(self) -> "Item":
+    def create_filler(self, reason: FillerReason = FillerReason.undefined) -> "Item":
         return self.create_item(self.get_filler_item_name())
```

Comment:
We say people shouldn't need to override this method, but we don't do anything with the variable here.

Either:
* Remove the above comment
* Pass the reason into get_filler_item_name

My preference is the latter, but it will 100% be a major breaking change to do so